<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QwtDial" name="pskPhaseDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pskIFdisplay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pskQualitydisplay">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="pskAfconTrigger">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Afc off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Afc on</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="pskReverseTrigger">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>reverse</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="pskModeTrigger">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>psk31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>psk63</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>qpsk31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>qpsk63</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>psk125</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>qpsk125</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="pskSquelchLevelTrigger">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="pskFilterDegreeTrigger">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="value">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="psktextBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>psk decoder</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
